Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/configuration/server/CCLConfigurationServer.kt

```diff
@@ -39,7 +39,7 @@ class CCLConfigurationServer @Inject constructor(
             }
         } catch (e: Exception) {
             Timber.tag(TAG).e(e, "Failed to get ccl configuration")
-            null
+            throw e
         }
     }.also { Timber.tag(TAG).d("Returning %s", it) }
```

Comment:
Using `ByteArray` parameter in the log seems to be useless as it's not translated into a string. It could be removed, not necessary in this PR, as this is about something else.

Reply:
You are right. I can remove it with this PR. Is that fine for you @mtwalli ?

Reply:
Yes, makes sense